crypto-extensions/src/main/java/net/corda/v5/crypto/extensions/merkle/MerkleTreeHashDigestProviderWithSizeProofSupport.java

```diff
@@ -0,0 +1,10 @@
+package net.corda.v5.crypto.extensions.merkle;
+
+import net.corda.v5.crypto.merkle.MerkleProof;
+import org.jetbrains.annotations.NotNull;
+import java.util.List;
+
+public interface MerkleTreeHashDigestProviderWithSizeProofSupport extends MerkleTreeHashDigestProvider {
```

Comment:
This needs a JDOC.

Reply:
I always thought one interface or class per file was strongly encouraged (or even required) as good style in Java, and that's something Kotlin changes.